catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/image/ImageAttribute.java

```diff
@@ -30,189 +31,45 @@
-                    .getSecurityClassificationSystem()),
-    IMAGE_CODEWORDS("imageCodewords",
-            "ISCODE",
-            segment -> segment.getSecurityMetadata()
-                    .getCodewords()),
-    IMAGE_CONTROL_AND_HANDLING("imageControlandHandling",
-            "ISCTLH",
-            segment -> segment.getSecurityMetadata()
-                    .getControlAndHandling()),
-    IMAGE_RELEASING_INSTRUCTIONS("imageReleasingInstructions",
-            "ISREL",
-            segment -> segment.getSecurityMetadata()
-                    .getReleaseInstructions()),
-    IMAGE_DECLASSIFICATION_TYPE("imageDeclassificationType",
-            "ISDCTP",
-            segment -> segment.getSecurityMetadata()
-                    .getDeclassificationType()),
-    IMAGE_DECLASSIFICATION_DATE("imageDeclassificationDate",
-            "ISDCDT",
-            segment -> segment.getSecurityMetadata()
-                    .getDeclassificationDate()),
-    IMAGE_DECLASSIFICATION_EXEMPTION("imageDeclassificationExemption",
-            "ISDCXM",
-            segment -> segment.getSecurityMetadata()
-                    .getDeclassificationExemption()),
-    IMAGE_DOWNGRADE("imageDowngrade",
-            "ISDG",
-            segment -> segment.getSecurityMetadata()
-                    .getDowngrade()),
-    IMAGE_DOWNGRADE_DATE("imageDowngradeDate",
-            "ISDGDT",
-            segment -> segment.getSecurityMetadata()
-                    .getDowngradeDate()),
-    IMAGE_CLASSIFICATION_TEXT("imageClassificationText",
-            "ISCLTX",
-            segment -> segment.getSecurityMetadata()
-                    .getClassificationText()),
-    IMAGE_CLASSIFICATION_AUTHORITY_TYPE("imageClassificationAuthorityType",
-            "ISCATP",
-            segment -> segment.getSecurityMetadata()
-                    .getClassificationAuthorityType()),
-    IMAGE_CLASSIFICATION_AUTHORITY("imageClassificationAuthority",
-            "ISCAUT",
-            segment -> segment.getSecurityMetadata()
-                    .getClassificationAuthority()),
-    IMAGE_CLASSIFICATION_REASON("imageClassificationReason",
-            "ISCRSN",
-            segment -> segment.getSecurityMetadata()
-                    .getClassificationReason()),
-    IMAGE_SECURITY_SOURCE_DATE("imageSecuritySourceDate",
-            "ISSRDT",
-            segment -> segment.getSecurityMetadata()
-                    .getSecuritySourceDate()),
-    IMAGE_SECURITY_CONTROL_NUMBER("imageSecurityControlNumber",
-            "ISCTLN",
-            segment -> segment.getSecurityMetadata()
-                    .getSecurityControlNumber()),
-    IMAGE_SOURCE("imageSource", "ISORCE", ImageSegment::getImageSource),
-    NUMBER_OF_SIGNIFICANT_ROWS_IN_IMAGE("numberOfSignificantRowsInImage",
+    TARGET_IDENTIFIER(Isr.TARGET_ID,
+            "TGTID",
+            segment -> getTargetId(segment)),
+    IMAGE_IDENTIFIER_2(Isr.IMAGE_ID,
+            "IID2",
```

Comment:
Not for anyone wondering, it appears by convention that IID2 is the field where the "normal" 40 character image ID is stored.  IID2 appears to be used for other identifying information which has less value to end users.